src/modules/logger/logger.cpp

```diff
@@ -609,15 +614,23 @@ void Logger::add_default_topics()
 	add_topic("vehicle_status");
 }
 
-void Logger::add_calibration_topics()
+void Logger::add_thermal_calibration_topics()
 {
 	// Note: try to avoid setting the interval where possible, as it increases RAM usage
-
 	add_topic("sensor_gyro", 100);
 	add_topic("sensor_accel", 100);
 	add_topic("sensor_baro", 100);
 }
 
+void Logger::add_system_identification_topics()
+{
+	// for system id need to log imu and controls at full rate
+	add_topic("sensor_gyro");
+	add_topic("sensor_accel");
+	add_topic("actuator_controls_0");
```

Comment:
Please feel free to push changes to this branch. I won't have time to touch it for a day or 2, then I can come back and review.